bootstrap-salt.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh -
+/#!/bin/sh -
```

Comment:
Was this additional slash intentional? ;)

Reply:
Sorry, my bad. Fixed.